app/models/manageiq/providers/openstack/refresh_parser_common/helper_methods.rb

```diff
@@ -52,22 +52,22 @@ def safe_call
           yield
         rescue Excon::Errors::Forbidden => err
           # It can happen user doesn't have rights to read some tenant, in that case log warning but continue refresh
-          _log.warn "Forbidden response code returned in provider: #{@os_handle.address}. Message=#{err.message}"
+          _log.warn "Forbidden response code returned in provider: #{@os_handle&.address}. Message=#{err.message}"
```

Comment:
@aufi could this be `manager.hostname` instead of `@os_handle.address` ?

Reply:
Yep, manager makes more sense. Since it is a helper module and the code is in rescue block, I prefer using instance variable to not fail on undefined method `manager` in any case (but the manager and @manager are accessible in normal conditions)

Reply:
:+1:

Reply:
`@manager` should be set by the Collector initialize so I figured it was more likely to be present than `@os_handle`